src/compiler/log_global_dependencies.ts

```diff
@@ -0,0 +1,21 @@
+import { writeFile } from 'fs/promises';
+import { join } from 'path';
+
+import { execSyncPretty } from './utils/exec_sync_pretty';
+import { AZLE_PACKAGE_PATH } from './utils/global_paths';
+
+export async function logGlobalDependencies(): Promise<void> {
+    const wasiVersion = execSyncPretty('cargo install --list | grep wasi2ic');
+    const nodeVersion = execSyncPretty('node --version');
+    const rustVersion = execSyncPretty('rustc --version');
+
+    const globalDependencies = `WASI2IC Version: ${wasiVersion}
```

Comment:
Can you lowercase all of the names here? And lowercase version? And saw rustc instead of Rust

Reply:
sure